packages/storage/src/database/general/database.ts

```diff
@@ -58,11 +61,15 @@ export class ModelDatabase<T extends Document> extends ModelBasic<T> {
   public async createCollection(
     collectionName: string,
     session?: ClientSession
-  ): Promise<void> {
+  ): Promise<boolean> {
     const isExist = await this.isCollectionExist(collectionName);
```

Comment:
this `isCollectionExist` from `ModelDatabase` should be index as well. The `ModelDatabase` that manage these collection behaviour kinda mess, you should check it